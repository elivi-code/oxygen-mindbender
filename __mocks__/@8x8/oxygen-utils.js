export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
