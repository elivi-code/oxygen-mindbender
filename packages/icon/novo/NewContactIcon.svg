<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M3 15V12.75H2.25C1.83579 12.75 1.5 12.4142 1.5 12C1.5 11.5858 1.83579 11.25 2.25 11.25H3V9H2.25C1.83579 9 1.5 8.66421 1.5 8.25C1.5 7.83579 1.83579 7.5 2.25 7.5H3V6C3 3.51472 5.01472 1.5 7.5 1.5H18C20.4853 1.5 22.5 3.51472 22.5 6V12.75C22.5 13.1642 22.1642 13.5 21.75 13.5C21.3358 13.5 21 13.1642 21 12.75V6C21 4.34315 19.6569 3 18 3H7.5C5.84315 3 4.5 4.34315 4.5 6V18C4.5 19.6569 5.84315 21 7.5 21H12.75C13.1642 21 13.5 21.3358 13.5 21.75C13.5 22.1642 13.1642 22.5 12.75 22.5H7.5C5.01472 22.5 3 20.4853 3 18V16.5H2.25C1.83579 16.5 1.5 16.1642 1.5 15.75C1.5 15.3358 1.83579 15 2.25 15H3Z" fill="#292929"/>
<path d="M16.1371 14.7753C16.4444 14.5119 16.4854 14.045 16.1657 13.7968C15.3404 13.1559 14.1163 12.75 12.75 12.75C10.2647 12.75 8.25 14.0931 8.25 15.75C8.25 17.2368 9.87232 18.4709 12.0012 18.7087C12.4129 18.7546 12.75 18.4142 12.75 18C12.75 17.5858 12.4123 17.2564 12.0024 17.1968C11.3453 17.1012 10.7941 16.8859 10.4001 16.6232C9.85391 16.2591 9.75 15.9209 9.75 15.75C9.75 15.5791 9.85391 15.2409 10.4001 14.8768C10.9357 14.5196 11.7622 14.25 12.75 14.25C13.7378 14.25 14.5643 14.5196 15.0999 14.8768C15.3753 15.0604 15.7614 15.0974 16.0127 14.882L16.1371 14.7753Z" fill="#292929"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M12.75 11.25C14.4069 11.25 15.75 9.90685 15.75 8.25C15.75 6.59315 14.4069 5.25 12.75 5.25C11.0931 5.25 9.75 6.59315 9.75 8.25C9.75 9.90685 11.0931 11.25 12.75 11.25ZM12.75 9.75C13.5784 9.75 14.25 9.07843 14.25 8.25C14.25 7.42157 13.5784 6.75 12.75 6.75C11.9216 6.75 11.25 7.42157 11.25 8.25C11.25 9.07843 11.9216 9.75 12.75 9.75Z" fill="#292929"/>
<path d="M18.75 15C18.3358 15 18 15.3358 18 15.75V18H15.75C15.3358 18 15 18.3358 15 18.75C15 19.1642 15.3358 19.5 15.75 19.5H18V21.75C18 22.1642 18.3358 22.5 18.75 22.5C19.1642 22.5 19.5 22.1642 19.5 21.75V19.5H21.75C22.1642 19.5 22.5 19.1642 22.5 18.75C22.5 18.3358 22.1642 18 21.75 18H19.5V15.75C19.5 15.3358 19.1642 15 18.75 15Z" fill="#292929"/>
</svg>
