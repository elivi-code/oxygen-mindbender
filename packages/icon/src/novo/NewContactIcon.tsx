import React from 'react';
import type { IconProps } from '../utils/createSvgIcon';
import createSvgIcon from '../utils/createSvgIcon';

export type { IconProps };
export default createSvgIcon(<g><path d="M3 15v-2.25h-.75a.75.75 0 0 1 0-1.5H3V9h-.75a.75.75 0 0 1 0-1.5H3V6c0-2.4853 2.0147-4.5 4.5-4.5H18c2.4853 0 4.5 2.0147 4.5 4.5v6.75a.75.75 0 0 1-1.5 0V6c0-1.6569-1.3431-3-3-3H7.5c-1.6569 0-3 1.3431-3 3v12c0 1.6569 1.3431 3 3 3h5.25a.75.75 0 0 1 0 1.5H7.5C5.0147 22.5 3 20.4853 3 18v-1.5h-.75a.75.75 0 0 1 0-1.5H3Z"/><path d="M16.1371 14.7753c.3073-.2634.3483-.7303.0286-.9785-.8253-.6409-2.0494-1.0468-3.4157-1.0468-2.4853 0-4.5 1.3431-4.5 3 0 1.4868 1.6223 2.7209 3.7512 2.9587.4117.0459.7488-.2945.7488-.7087s-.3377-.7436-.7476-.8032c-.6571-.0956-1.2083-.3109-1.6023-.5736-.5462-.3641-.6501-.7023-.6501-.8732 0-.1709.104-.5091.6501-.8732.5356-.3572 1.3621-.6268 2.3499-.6268.9878 0 1.8143.2696 2.3499.6268.2754.1836.6615.2206.9128.0052l.1244-.1067Z"/><path fillRule="evenodd" d="M12.75 11.25c1.6569 0 3-1.3431 3-3s-1.3431-3-3-3-3 1.3431-3 3 1.3431 3 3 3Zm0-1.5c.8284 0 1.5-.6716 1.5-1.5s-.6716-1.5-1.5-1.5-1.5.6716-1.5 1.5.6716 1.5 1.5 1.5Z" clipRule="evenodd"/><path d="M18.75 15a.75.75 0 0 0-.75.75V18h-2.25a.75.75 0 0 0 0 1.5H18v2.25a.75.75 0 0 0 1.5 0V19.5h2.25a.75.75 0 0 0 0-1.5H19.5v-2.25a.75.75 0 0 0-.75-.75Z"/></g>, 'NewContactIcon');
