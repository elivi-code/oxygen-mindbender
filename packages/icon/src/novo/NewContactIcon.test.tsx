/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import NewContactIcon from './NewContactIcon';

describe('<NewContactIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<NewContactIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
