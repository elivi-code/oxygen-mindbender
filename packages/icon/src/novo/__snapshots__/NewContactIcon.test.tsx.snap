// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewContactIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <path
      d="M3 15v-2.25h-.75a.75.75 0 0 1 0-1.5H3V9h-.75a.75.75 0 0 1 0-1.5H3V6c0-2.4853 2.0147-4.5 4.5-4.5H18c2.4853 0 4.5 2.0147 4.5 4.5v6.75a.75.75 0 0 1-1.5 0V6c0-1.6569-1.3431-3-3-3H7.5c-1.6569 0-3 1.3431-3 3v12c0 1.6569 1.3431 3 3 3h5.25a.75.75 0 0 1 0 1.5H7.5C5.0147 22.5 3 20.4853 3 18v-1.5h-.75a.75.75 0 0 1 0-1.5H3Z"
    />
    <path
      d="M16.1371 14.7753c.3073-.2634.3483-.7303.0286-.9785-.8253-.6409-2.0494-1.0468-3.4157-1.0468-2.4853 0-4.5 1.3431-4.5 3 0 1.4868 1.6223 2.7209 3.7512 2.9587.4117.0459.7488-.2945.7488-.7087s-.3377-.7436-.7476-.8032c-.6571-.0956-1.2083-.3109-1.6023-.5736-.5462-.3641-.6501-.7023-.6501-.8732 0-.1709.104-.5091.6501-.8732.5356-.3572 1.3621-.6268 2.3499-.6268.9878 0 1.8143.2696 2.3499.6268.2754.1836.6615.2206.9128.0052l.1244-.1067Z"
    />
    <path
      clipRule="evenodd"
      d="M12.75 11.25c1.6569 0 3-1.3431 3-3s-1.3431-3-3-3-3 1.3431-3 3 1.3431 3 3 3Zm0-1.5c.8284 0 1.5-.6716 1.5-1.5s-.6716-1.5-1.5-1.5-1.5.6716-1.5 1.5.6716 1.5 1.5 1.5Z"
      fillRule="evenodd"
    />
    <path
      d="M18.75 15a.75.75 0 0 0-.75.75V18h-2.25a.75.75 0 0 0 0 1.5H18v2.25a.75.75 0 0 0 1.5 0V19.5h2.25a.75.75 0 0 0 0-1.5H19.5v-2.25a.75.75 0 0 0-.75-.75Z"
    />
  </g>
</svg>
`;
